notifications/admin.py

```diff
@@ -17,9 +20,35 @@ def __init__(self, *args, **kwargs):
         choices = [x for x in self.fields['type'].choices if x[0] not in existing_types]
         self.fields['type'].choices = choices
 
+        admins_qs = get_user_model().objects.filter(email__isnull=False).\
```

Comment:
I don't think `User.email` can be `null`, so this should be `.exclude(email='')`.